bleak/backends/scanner.py

```diff
@@ -307,6 +307,15 @@ def set_scanning_filter(self, **kwargs) -> None:
         """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    async def request_permissions(self, **kwargs) -> None:
```

Comment:
Since this only applies to Android, I don't think we should make this a common function to all platforms.

Also, it is not specifically related to the `BleakScanner`, so should not be a method on that object.

Making it a standalone function in `p2andoid.utils` seems like the logical place to put it.

Reply:
done